.github/workflows/pr_epm.yml

```diff
@@ -141,6 +141,7 @@ jobs:
     - name: Install centos dependency
       if: ${{ contains(matrix.tag, 'centos') }}
       run: docker exec $inclavare_test bash -c 'cd /root/inclavare-containers/${{ matrix.tag }};
```

Comment:
And plz add the comment to explain the operation of stopping dnf-makecache.timer.